joomla/joomla-cms
# [4.0] Reorganise administrator menu items - extensions manager

#### Summary of Changes

Reorganised the menu items under the extension manager top level menu item. As seen in the past few release we are no longer supporting Joomla core update via extensions manager, that is a good move. Having a good explanation from @brianteeman and @mbabker I was convinced well that these two need to operate independently with no internal handovers.

However, I can understand the annoyance caused to Brian and Michael explaining this to everyone separately over and again. I suggest to make some visual change that calls for the users' action.

With this reorganisation, people would feel that there is some change right there. Investing little they would be able to see both options alongside and tend more towards using appropriate option rather than assuming "I did it before, why not now!".
#### Testing Instructions
* **New testers only**: Please follow the testing instructions at [Testing Joomla! patches](https://docs.joomla.org/Testing_Joomla!_patches)
* Every link should open the correct page.
* Changing permissions on 'com_joomlaupdate', 'com_postinstall' and 'com_installer' should hide the respective menu items correctly.

~~PS: Asking for password on Joomla Update seems unnecessary and annoying as well. If we really want to be sure it could be just a confirm checkbox or a button.~~ Irrelevant here.
